### ISSUE TITLE
dnspython deprecation warning

### ISSUE THREAD

**Issue by sergray** [abusix/querycontacts#2]
dnspython 2.0 deprecated `Resolver.query`  method, which breaks applications, if deprecations are turned into the errors.

How to reproduce:

* download [test_querycontacts.py](https://gist.github.com/sergray/91f9eb4ca93340d893e36cf007926977)
* then run

```
$ virtualenv qce
$ source qce/bin/activate
$ pip install querycontacts
$ pip freeze -l
dnspython==2.0.0
ipaddr==2.2.0
querycontacts==1.1.1
$ python test_querycontacts.py 
sys.version: 3.8.6 (default, Sep 25 2020, 21:22:01) 
[GCC 7.5.0]
<module 'querycontacts' from '$HOME/qce/lib/python3.8/site-packages/querycontacts/__init__.py'> 1.1.1
<module 'dns' from '$HOME/qce/lib/python3.8/site-packages/dns/__init__.py'> 2.0.0
$ python -W error test_querycontacts.py 
sys.version: 3.8.6 (default, Sep 25 2020, 21:22:01) 
[GCC 7.5.0]
<module 'querycontacts' from '$HOME/qce/lib/python3.8/site-packages/querycontacts/__init__.py'> 1.1.1
<module 'dns' from '$HOME/qce/lib/python3.8/site-packages/dns/__init__.py'> 2.0.0
Traceback (most recent call last):
  File "test_querycontacts.py", line 8, in <module>
    querycontacts.ContactFinder().find(socket.gethostbyname('google.nl'))
  File "$HOME/qce/lib/python3.8/site-packages/querycontacts/__init__.py", line 62, in find
    contacts = self._get_txt_record(lookup)
  File "$HOME/qce/lib/python3.8/site-packages/querycontacts/__init__.py", line 68, in _get_txt_record
    answers = self.resolver.query(name, 'TXT')
  File "$HOME/qce/lib/python3.8/site-packages/dns/resolver.py", line 1087, in query
    warnings.warn('please use dns.resolver.Resolver.resolve() instead',
DeprecationWarning: please use dns.resolver.Resolver.resolve() instead
```

If `dnspython` is downgraded, the problem disappears:
```
$ pip install dnspython\<2.0
$ python -W error test_querycontacts.py
```

**Comment by FrederikP** [comment]
Hi, thanks for bringing this to our attention and opening up a PR. I will take a look

**Comment by FrederikP** [comment]
Released v 2.0.0 dropping support for python 2.7 and including your fix.
https://pypi.org/project/querycontacts/2.0.0/

Thanks for your contribution. Please let us know if anything isn't working for you.
Cheers!

**Comment by sergray** [comment]
@FrederikP thanks a lot for the prompt response and such quick update of the package :+1:
